include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -33,53 +33,57 @@
 #include <vector>
 
 namespace glow {
-/// Loads tensor \p T from the input \p in.
-inline llvm::Error loadWeight(const onnxTensorDescriptorV1 &in, Tensor *T) {
+
+// Result of loadWeight function.
+struct LoadWeightResult {
+  // Main Glow tensor loaded from the onnxTensorDescriptorV1, this is always
+  // non-null.
+  std::unique_ptr<Tensor> t;
+  // Glow tensor containing quantization biases. This should only be non-null if
+  // there is more than 1 quantization parameter in the onnxTensorDescriptorV1.
+  std::unique_ptr<Tensor> biases;
```

Comment:
Interesting choice of using Tensor over std::vector. You want to support mulit-type?

Reply:
Glow's existing nodes with multiple quantization params such as `RowwiseQuantizedFullyConnected` use Tensor inputs so I just stuck with that approach.